data/crac-file/crac-api/src/main/java/com/farao_community/farao/data/crac_api/Crac.java

```diff
@@ -19,33 +20,78 @@
 @JsonTypeInfo(use = JsonTypeInfo.Id.MINIMAL_CLASS)
 public interface Crac extends Identifiable, Synchronizable {
 
-    List<Cnec> getCnecs();
+    // Instants management
+    Instant getInstant(String id);
 
-    void setCnecs(List<Cnec> cnecs);
+    void addInstant(Instant instant);
 
-    List<RangeAction> getRangeActions();
+    // Contingencies management
+    Set<Contingency> getContingencies();
 
-    void setRangeActions(List<RangeAction> rangeActions);
+    Contingency getContingency(String id);
 
-    List<NetworkAction> getNetworkActions();
+    void addContingency(Contingency contingency);
 
-    void setNetworkActions(List<NetworkAction> networkActions);
+    //States management
+
+    /**
+     * Select the preventive state. This state must be unique. It's the only state that is
+     * defined with no contingency.
+     *
+     * @return The preventive state of the problem definition.
+     */
+    State getPreventiveState();
+
+    /**
+     * Chronological list of states after a defined contingency. The chronology is defined by
+     * instants objects.
+     *
+     * @param contingency: The contingency after which we want to gather states.
+     * @return Ordered list of states after the specified contingency.
+     */
+    List<State> getStates(Contingency contingency);
+
+    /**
+     * Unordered set of States defined at the same instant. It will be either the preventive state either
```

Comment:
It is "either .. or" in English, not "either .. either" :smiley_cat: 
(so "It will be either the preventive state OR ...")